docs/reference-architectures/sap/index.md

```diff
@@ -0,0 +1,418 @@
+be connected to their own subnet and associated with a network security group
+within the group. For implementation details, see [Filter network traffic with network security groups][filter-network].
+
+### Azure Storage
+With all database server virtual machines, we recommend using Azure Premium
+Storage for consistent read/write latency. For SAP application servers, including
+the (A)SCS virtual machines, you can use Azure Standard Storage, because
+application execution takes place in memory and uses disks for logging only.
+
+Each Azure blob has an IOPS quota. The common practices in storage volume
+performance optimization applies to Azure storage layout. For example, striping
+multiple disks together to create a larger disk volume improves IO performance.
+Enabling the read cache on storage content that changes infrequently enhances
+the speed of data retrieval.
+
+For the backup data store, we recommend using Azure [cool blob storage][cool-blob-storage].
+The cool storage tier is a cost-effective way to store data that is less
+frequently accessed and long-lived.
+
+For simplicity, we recommend using [Azure Managed Disks][managed-disks] to manage the 
+storage associated with the virtual machine disks. For the entire
+deployment, use Premium storage for all disk performance-dependent workloads
+
+We recommend using Managed Disks for all layers of the SAP application stack.
+The Managed Disks service makes it much simpler to deploy your virtual machines.
+For example, you no longer need to set up additional storage accounts per
+virtual machine to handle limits (such as 20,000 IOPS per account), because
+Managed Disks handles storage behind the scenes. Managed Disks also makes sure
+the disks of virtual machines in an availability set are sufficiently isolated
+from each other to avoid single points of failure.
+
+## Scalability considerations
+At the SAP application layer, Azure offers a wide range of virtual machine sizes
+for scaling up and scaling out. Please see SAP note 1928533 for an inclusive
+list.
+
+For SAP HANA on Azure virtual machines with both OLTP and OLAP SAP applications,
+the SAP-certified virtual machine size is GS5 single instance. As larger virtual
+machines become available, you can scale up with the same cloud deployment. For
+larger workloads, Microsoft also offers [Azure Large Instances][azure-large-instances] 
+for SAP HANA on physical servers co-located in a Microsoft Azure certified
+datacenter, which provides up to 4 TB of memory capacity for a single instance
+at this time. Multi-node configuration is also possible with a total memory
+capacity of up to 32 TB.
+
+## Availability considerations
+The database tier's native system replication feature provides either manual or automatic failover between replicated nodes. To
+further lower unplanned downtime on Linux-based database systems, a high availability extension for the specific
+Linux distribution is required.
+
+## Disaster recovery considerations
```

Comment:
We should follow the existing RA outline if possible. Can we fold this under Availability?